<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntiPlag</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTextEdit" name="textEdit1"/>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btCompare">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare Texts</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCdnce">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Percentage:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="coincidenceBar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>211</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Plagarized Words:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listOfWords"/>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>KMP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton2">
       <property name="text">
        <string>Rabin-Karp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton3">
       <property name="text">
        <string>Boyer-Moore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton5">
       <property name="text">
        <string>Hamming Distance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton4">
       <property name="text">
        <string>Brute-Force</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>892</x>
      <y>447</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Documents</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check for Plagarizm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1045</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
